02_selecting_gene_ids/select_gene_ids.py

```diff
@@ -0,0 +1,107 @@
+"""
+For each Illumina platform that we support:
+  - Read the dataframe of duplicate probes and the different gene IDs
+  - Read the brainarray occurrence counts for the current species
+  - Group the duplicate probes dataframe by probe ID
+  - For each group, pick the gene ID with the highest occurrence count. If there
+    are ties, then we want to pick the gene ID that is on the main Ensembl
+    assembly. If there are multiple genes on the main assembly, pick the one with
```

Comment:
> If there are ties, then we want to pick the gene ID that is on the main Ensembl assembly

This isn't reflected in the section of the readme that outlines the priority.

Reply:
Good catch; this was our original decision for priority but this didn't end up actually breaking any ties and the implementation would not have been straightforward, so I removed that check. I will update this comment.